app/src/main/java/com/belatrixsf/allstars/ui/common/views/searchingview/SearchingViewPresenter.java

```diff
@@ -36,6 +38,10 @@ public void searchingTextTyped(String searchingText) {
         searchableView.notifyTextTyped(searchingText);
     }
 
+    public void editorAction(View view, int actionId){
+        searchableView.editorAction(view, actionId);
+    }
```

Comment:
we can't do this. having a view is having an android related class on the presenter